### ISSUE TITLE
Support for RML-star?

### ISSUE THREAD

**Issue by jetztgradnet** [carml/carml#381]
Hi, 
we are using Carml for our implementation of an ETL pipeline for RDF. For some use cases it would be desirable to generate RDF-star as part of a conversion.

Are there any plans to implement RML-star to generate RDF-star?

See these papers and articles from the community group:

* section 5.5 in [The RML Ontology: A Community-Driven Modular Redesign After a Decade of Experience in Mapping Heterogeneous Data to RDF](https://link.springer.com/content/pdf/10.1007/978-3-031-47243-5_9.pdf)
* https://biblio.ugent.be/publication/8724424/file/8724427
* https://www.semantic-web-journal.net/system/files/swj3602.pdf

**Comment by pmaria** [comment]
@jetztgradnet yes, definitely. I'm planning to add support for all the CG specs in the coming months. I'll update this issue once RML-star is supported. This will most probably be a new major release,

**Comment by jetztgradnet** [comment]
That was fast ;-) 
Let me know if you need somebody to test this.

**Comment by davidlamprecht** [comment]
Hey @pmaria ,
can you already give a brief update about the current status of the upcoming RML-star / RDF-star extension?

**Comment by davidlamprecht** [comment]
Hi @pmaria ,

Thanks for the update. Please let us know once the RML-star features are testable. Then we can support you with that.
